tensorflow/compiler/aot/tfcompile.bzl

```diff
@@ -145,7 +145,7 @@ def tf_library(name, graph, config,
            " --target_triple=" + target_llvm_triple() +
            " --out_header=$(@D)/" + header_file +
            " --out_object=$(@D)/" + object_file +
-           " " + (tfcompile_flags or "")),
+           " ".join(tfcompile_flags or [])),
```

Comment:
Just like `execv()` having a list of arguments implies that those arguments don't need to be bash syntax. According to the [bash documentation](https://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html#Single-Quotes), the correct way to escape these values is probably:

```py
flags = " ".join(["'" + arg.replace("'", "'\\''") + "'" for arg in tfcompile_flags])
```

In other words, `foo` becomes `'foo'` and `foo'bar` becomes `'foo'\''bar'`.